services-js/commissions-search/src/index.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import App from "./App";
+import registerServiceWorker from "./registerServiceWorker";
+
+import { ApolloProvider } from "react-apollo";
+import { ApolloClient } from "apollo-client";
+import { InMemoryCache } from "apollo-cache-inmemory";
+import { createHttpLink } from "apollo-link-http";
+
+const client = new ApolloClient({
+  link: new createHttpLink({
+    uri:
+      "https://commissions-app.digital-staging.boston.gov/commissions/graphql",
```

Comment:
You'll need a way of parameterizing this, and also including an API key.

Reply:
@finneganh can we use environment variables?

Reply:
I think your best bet here might be to have this set to staging, but pull off of `window.COMMISSIONS_APP_GRAPHQL_ENDPOINT` or something if it exists. Then you can set that from a variable on the Drupal side.

The config is a little tricky because I think we're just going to serve this as static files out of S3.

You could alternately try to make it a configuration at build time? Webpack can set variables.

Reply:
put value on window